apps/federatedfilesharing/src/components/PersonalSettings.vue

```diff
@@ -29,19 +29,19 @@
 					<Facebook :size="20" />
 				</template>
 			</NcButton>
-			<NcButton @click="goTo(shareTwitterUrl)">
-				{{ t('federatedfilesharing', 'Twitter') }}
+			<NcButton @click="goTo(shareXUrl)">
+				{{ t('federatedfilesharing', 'Formerly Twitter') }}
```

Comment:
```suggestion
				{{ t('federatedfilesharing', 'X (formerly Twitter)') }}
```
I think it is preferred to have it this way (or just remove the entire Twitter part).

Reply:
I think keeping the twitter part is better. Since X is not really a word imo

Reply:
Agreed, just shows this rename was stupid